src/test/suite/fixtures/recorded/languages/rust/changeNameDrum2.yml

```diff
@@ -0,0 +1,38 @@
+languageId: rust
```

Comment:
this should also work for "key", as it looks a lot like a key-value pair

Fwiw we're planning to support "name" everywhere we support "key" in the future